Adding docs for dapr init changes (#2295)
dapr/docs: daprdocs/content/en/operations/support/support-preview-features.md

| ---------- |-------------|---------|---------------|
| **Partition actor reminders** | Allows actor reminders to be partitioned across multiple keys in the underlying statestore in order to improve scale and performance. | `Actor.TypeMetadata` | [How-To: Partition Actor Reminders]({{< ref "howto-actors.md#partitioning-reminders" >}}) |
| **Pub/Sub routing** | Allow the use of expressions to route cloud events to different URIs/paths and event handlers in your application. | `PubSub.Routing` | [How-To: Publish a message and subscribe to a topic]({{<ref howto-route-messages>}}) |
| **ARM64 Mac Support** | Dapr CLI, sidecar, and Dashboard are now natively compiled for ARM64 Macs, along with Dapr CLI installation via Homebrew. | N/A | [Install the Dapr CLI]({{<ref install-dapr-cli>}}) |

> | **--image-registry** flag with Dapr CLI| In standalone mode we can specify this flag to specify any private registry to pull various container images required to install Dapr| N/A | [init CLI command reference]({{<ref "dapr-init.md#self-hosted-environment" >}}) |

Reviewer: ```suggestion
| **--image-registry** flag with Dapr CLI| In self hosted mode you can set this flag to specify any private registry to pull the container images required to install Dapr| N/A | [init CLI command reference]({{<ref "dapr-init.md#self-hosted-environment" >}}) |
```

Reviewer: Given that this has been set as a preview feature in the CLI and therefore does not need runtime configuration, can you change this paragraph at the start of this article

Preview features in Dapr are considered experimental when they are first released.

Runtime review features require explicit opt-in in order to be used. The runtime opt-in is specified in a preview `setting` feature in Dapr's application configuration. See [How-To: Enable preview features]({{}}) for information more information.

For CLI there is no explicit opt-in, just the version that this was first made available.

Reviewer: Can you also add a new column to the preview feature table called "Version introduced".
And for the 4 remain preview features these are the version release
Actor partitioning v1.4
Mac ARM64 v1.5
Pub/Sub routing v1.7
image respository v1.7
